Python_and_Git/Task_1_num.py

```diff
@@ -0,0 +1,11 @@
+num = 15
+num
+num = num + 5
+num
+num = num - 3
+num
+num *= 4
+num
+num //= 2
+num
+print (num)
```

Comment:
Please do not add redundant variables in `.py` files
```suggestion
num = 15
num += 5
num -= 3
num *= 4
num //= 2
print(num)
```